src/main.ts

```diff
@@ -181,6 +184,11 @@ const createWindow = async () => {
   configureWatcher(mainWindow)
   wasAppClosedByClient = false
 
+  proxyEmitter.on('status:change', (status: ProxyStatus) => {
```

Comment:
Thanks @Llandy3d for pointing this out in [this PR](https://github.com/grafana/k6-studio/pull/513/files#r1976740059)